tetris.js

```diff
@@ -452,5 +452,47 @@ module.exports = {
 				return this.coordinates
 			}
 		}
+	},
+	Field: class {
+		constructor(field) {
+			if (Array.isArray(field)) {
+				this.field = field
+			} else {
+				this.field = []
+				for (let i = 0; i < field.y; i++) {
+					let array = []
+					for (let j = 0; j < field.x; j++) {
+						array.push(false)
+					}
+					this.field.push(array)
+				}
+			}
+		}
+		addTetrominoToField(coordinatesOfTetromino) {
+			for (let i = 0; i < coordinatesOfTetromino.length; i++) {
+				this.field[coordinatesOfTetromino[i].y][coordinatesOfTetromino[i].x] = true
+			}
+			return this.field
+		}
+		cleanFilledRows() {
+			let partOfTheField = []
+			for (let i = 0; i < this.field[0].length; i++) {
+				partOfTheField.push(false)
+			}
+			for (let i = 0; i < this.field.length; i++) {
+				let sumFullCells = 0
+				for (let j = 0; j < this.field[i].length; j++) {
+					if (this.field[i][j]) {
+						sumFullCells++
+					}
+				}
+				if (this.field.length === sumFullCells) {
+					this.field.splice(i, 1);
+					this.field.unshift(partOfTheField)
```

Comment:
this is not completely correct

Write tests that (remove lines, add tetrominos) - repeat several times. You will notice a bug